deployment/ccip/changeset/solana/cs_idl.go

```diff
@@ -0,0 +1,487 @@
+package solana
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/gagliardetto/solana-go"
+	"github.com/pelletier/go-toml"
```

Comment:
Did you choose v1 over v2 deliberately? I think we prefer v2 when possible.
```suggestion
	"github.com/pelletier/go-toml/v2"
```